pkg/secretless/entrypoint/entrypoint.go

```diff
@@ -35,13 +34,17 @@ type SecretlessOptions struct {
 func StartSecretless(params *SecretlessOptions) {
 	showVersion(params.ShowVersion)
 
-	verifyPlugins(params.PluginDir, params.PluginChecksumsFile)
-
 	// Construct the deps of Service
 	cfg := readConfig(params.ConfigFile)
 	logger := secretlessLog.New(params.DebugEnabled)
 	evtNotifier := eventnotifier.New(nil)
-	availPlugins := &proxyservice.AvailPluginStub{}
+	availPlugins, err := plugin.AllAvailablePlugins(params.PluginDir,
```

Comment:
nit: re consistent indentation choice...
there's still at least one instance of both:
```
plugin.AllAvailablePlugins(params.PluginDir,
	params.PluginChecksumsFile,...
```
and
```
plugin.AllAvailablePlugins(
	params.PluginDir,
	params.PluginChecksumsFile,...
```

Reply:
fixed for now - we can talk about the long-term approach on slack for this